catkin_ws/src/lidar_process/msg/tracked_obj.msg

```diff
@@ -0,0 +1,6 @@
+Header header
+uint32 object_id
+string object_type
+float32 pos_x 
```

Comment:
We could potentially replace the individual x,y,z with a single geometry_msgs/PointStamped[]

Reply:
instead of publishing one msg per object, the node is now publishing an array of objects. Please review